requirements.txt

```diff
@@ -0,0 +1,7 @@
+kenlm
```

Comment:
I don't think kenlm is available on PyPI, at least not with this name. Did you test this on a virtualenv that doesn't have all the requirements? In any case, we already have a requirements.txt file at [doc/requirements.txt](https://github.com/mozilla/DeepSpeech/blob/master/doc/requirements.txt). What you can do is move it from doc/ to the repository root so it's more visible, and I'll update the readthedocs.org setting accordingly.